content/en/docs/application-store/external-apps/deploy-tidb.md

```diff
@@ -0,0 +1,217 @@
+
+   {{</ notice >}} 
+
+7. Click **Deploy** and you can see two apps in the list as shown below:
+
+   ![tidb-cluster-app-running](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tidb-cluster-app-running.PNG)
+
+### Step 5: View TiDB cluster status
+
+1. Go to **Workloads** under **Application Workloads**, you can see all TiDB cluster Deployments are up and running.
+
+   ![tidb-cluster-deployments-running](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tidb-cluster-deployments-running.PNG)
+
+2. Switch to **StatefulSets** tab, you can see TiDB, TiKV and PD are up and running.
+
+   ![tidb-statefulsets](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tidb-statefulsets.PNG)
+
+   {{< notice note >}}
+
+   TiKV and TiDB will be created automatically and it may take a while before they display in the list.
+
+   {{</ notice >}}
+
+3. Click a single StatefulSet to go to its details page. You can see the metrics in line charts over a period of time under the **Monitoring** tab.
+
+   TiDB metrics:
+
+   ![tidb-metrics](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tidb-metrics.PNG)
+
+   TiKV metrics:
+
+   ![tikv-metrics](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tikv-metrics.PNG)
+
+   PD metrics:
+
+   ![pd-metrics](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/pd-metrics.PNG)
+
+4. Go to **Pods** under **Application Workloads**, you can see the TiDB cluster contains two TiDB Pods, three TiKV Pods, and three PD Pods.
+
+   ![tidb-pod-list](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tidb-pod-list.PNG)
+
+5. Go to **Volumes** under **Storage**, you can see TiKV and PD are using persistent volumes.
+
+   ![tidb-storage-usage](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tidb-storage-usage.PNG)
+
+6. Volume usage is also monitored. Click a volume item to go to the details page. Here is an example of TiKV:
+
+   ![tikv-volume-status](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tikv-volume-status.PNG)
+
+7. On the **Overview** page of the project, you can see a list of resource usage in the current project.
+
+   ![tidb-project-resource-usage](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tidb-project-resource-usage.PNG)
+
+### Step 6: Access the TiDB cluster
+
+1. Go to **Services** under **Application Workloads**, you can see detailed information of all services. As the service type is set to `NodePort` by default, you can access it through the Node IP address outside the cluster.
```

Comment:
Go to **Services** under **Application Workloads**, you can see detailed information of all services. As the service type is set to
->
Go to **Services** under **Application Workloads**, and you can see detailed information of all Services. As the Service type is set to